react/src/components/atoms/forms/InputCurrency/InputCurrency.knobs.options.js

```diff
@@ -0,0 +1,24 @@
+import { text, boolean, number, object, select } from '@storybook/addon-knobs/react';
+
+export default {
+  hiddenLabel: () => boolean('InputCurrency.hiddenLabel', false),
+  labelText: () => text('InputCurrency.labelText', 'Text Input'),
+  required: () => boolean('InputCurrency.required', false),
+  id: () => text('InputCurrency.id', 'text-input'),
+  name: () => text('InputCurrency.name', 'text-input'),
+  width: () => number('InputCurrency.width', 0),
+  maxlength: () => number('InputCurrency.maxlength', 20),
+  placeholder: () => text('InputCurrency.placeholder', 'type something'),
+  errorMsg: () => text('InputCurrency.errorMsg', 'you did not type something'),
+  defaultText: () => text('InputCurrency.defaultText', '0'),
+  max: () => number('InputCurrency.max', 1000),
```

Comment:
can we increas this to 10,000 for the story so users can see the comma sep formatting